src/utils/dateFormat.js

```diff
@@ -0,0 +1,14 @@
+function dateFormat(date) {
+  const formatDate = new Date(date);
+  const year = formatDate.getFullYear();
+  const month =
+    formatDate.getMonth() + 1 < 9
+      ? '0' + formatDate.getMonth() + 1
+      : formatDate.getMonth() + 1;
```

Comment:
padStart 을 사용하면 삼항연산자를 사용하지 않아도 0을 채울 수 있어요!

```suggestion
const month = String(formatDate.getMonth() + 1).padStart(2, '0')
```

Reply:
처음 알았습니다 😮
오늘도 하나 배워갑니다!